main/src/com/google/refine/model/Column.java

```diff
@@ -168,4 +231,23 @@ static public Column load(String s) throws Exception {
     public String toString() {
         return _name;
     }
+    
+    public <Any> Any castValue(String value)
+            throws InvalidCastException, ConstraintsException {
+        if (this.type.isEmpty()) {
+            throw new InvalidCastException();
+        } else {
+            try {
+                // Using reflection to invoke appropriate type casting method from the
+                // TypeInferrer class
+                String castMethodName = "cast" + (this.type.substring(0, 1).toUpperCase() + this.type.substring(1));
```

Comment:
It seems quite dangerous to do this if we allow types to be any string.

Reply:
why are Trys dangerous in Java ?  Sorry, I thought they avoid danger ? :)

Reply:
it's not the trys, it's just that the introspection features should be used only when needed, and this does not seem to be a case where they should be used. I would go for an enum for the type and a switch for the cast.

Reply:
That's part of the table scheme java library